optimum/intel/openvino/configuration.py

```diff
@@ -493,10 +483,13 @@ def post_init(self):
                 "quantization algorithm is selected and compression ratio is 1.0."
             )
 
+        if self.dtype in ["int4", "int8"]:
+            self.bits = 4 if self.dtype == "int4" else 8
+
```

Comment:
should we add a warning if `dtype` and `bits` were originally set to be incompatible to warn users ?